### DISCUSSION TITLE
Welcome to app Discussions!

### DISCUSSION THREAD

**Discussion by luanfreitasdev** [laradumps/app#78] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by senaszel** [comment]
Hi there, @luanfreitasdev Just letting you know there's 404 on URL to MacOs App. I couldn't reach your otherwise. Feel free to delete this message.
<img width="1440" alt="Screenshot 2023-04-23 at 23 00 49" src="https://user-images.githubusercontent.com/40730945/233866445-69c19e8d-3c58-43a5-9473-ef69fbccdf83.png">

**Reply by luanfreitasdev** [reply]
Thanks for watching! I redirected the user to another link

**Comment by Krato** [comment]
Hi. Is just me or we are unable to create new discussions? 😅
